frontend/packages/kubevirt-plugin/src/utils/basic.ts

```diff
@@ -0,0 +1,19 @@
+import * as _ from 'lodash';
+
+export const omitEmpty = (obj, justUndefined = false) => {
```

Comment:
Why `basic.ts` and not `common.ts` ? :smiley: 

Also, can we unit-test this? (covering at least the happy/expected execution paths)

Reply:
yes will do, the followup will be on the way soon